### DISCUSSION TITLE
Docs: Ability to Disable Rainbow Text

### DISCUSSION THREAD

**Discussion by joealden** [unocss/unocss#3601] [Ideas]
### Clear and concise description of the problem

This may only impact a small amount of people, but the constant color changes to links, code etc. on the docs site makes it borderline impossible to use for me (it's super distracting).

### Suggested solution

Add a button (like the dark/light mode button) to toggle this behaviour on/off.

### Alternative

Remove the text color change logic completely - it doesn't really add anything to the site.

### Additional context

I know that I could achieve this by enabling `prefers-reduced-motion` in my browser, but this isn't really what that setting is designed for, and you can only enable/disable that globally I believe, not on certain sites (this is the only site where this kind of motion impacts me).

### Validations

- [X] Read the [Contributing Guidelines](https://github.com/unocss/unocss/blob/main/CONTRIBUTING.md).
- [X] Read the `README.md` of using the package.
- [X] Already used the [Interactive Docs](https://unocss.dev/interactive/).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by LeoniePhiline** [comment]
I had considered choosing UnoCSS over TailwindCSS and started reading <https://unocss.dev/guide/why>.

I could not bear the color changes, as they were too distracting.

Unfortunately for me this means that the docs are not usable, and so is the entire library. I cannot learn using UnoCSS if I cannot focus on the text.

**Comment by userquin** [comment]
it should be deactivated if you enable prefer reduce motion: https://github.com/unocss/unocss/blob/main/docs/.vitepress/theme/rainbow.css#L90-L94

**Reply by joealden** [reply]
See original comment - already mentioned

**Reply by userquin** [reply]
so, what you need/want is adding some element in the ui to deactivate the rainbow, I'll check what we can do, stay tuned ;)

**Reply by joealden** [reply]
Exactly - thanks!

**Reply by userquin** [reply]
Cooking the new switcher...

![imagen](https://github.com/unocss/unocss/assets/6311119/e985361d-4521-448c-8aa3-874a3817a222)

**Reply by userquin** [reply]
here we go: https://github.com/unocss/unocss/pull/3931

**Reply by userquin** [reply]
@joealden you should be able to disable rainbow animation in the version navbar submenu
